example/reader_util.h

```diff
@@ -234,6 +235,12 @@ typedef struct ndpi_flow_info {
       char hostname[48];
       char fqdn[48];
     } softether;
+    struct {
+      char identity_uuid[36];
+      char machine[48];
+      char platform[48];
+      char services[64];
+    } tivoconnect;
```

Comment:
It is not critical, but it s probably better to align these sizes to the ones in `struct ndpi_flow_struct`